ehabi32/ehabi32.rst

```diff
@@ -1427,6 +1431,18 @@ The details are as follows:
 
     In general an unwinder must load *all* the machine registers listed in the VRS.
 
+   If Branch Target Identification mechanism is enabled, the unwinder
+   may elect to transfer control to the routine installed in VRS[r15]
+   via a jump instruction that requires the jump target to be
+   identified as a valid destination for indirect jumps.
+
+   .. note::
+
+    The intent is that the routine installed in VRS[r15] should start with a
+    ``bti`` instruction (or equivalent), when it might operate in an environment
```

Comment:
IMHO your phrase reads better if the comma is removed.

Reply:
Agree, it's even grammatically incorrect to place a comma in front of a dependent clause.